cdip_connector/core/schemas.py

```diff
@@ -314,7 +314,7 @@ class ERPatrolSegment(BaseModel):
     start_location: Optional[ERLocation]
     time_range: Optional[dict]
     updates: Optional[List[ERUpdate]]
-    track_points: Optional[List[ERObservation]]
+    track_points: Optional[List[ERObservation]] = []
```

Comment:
Can we change this to be:
```
    track_points: Optional[List[ERObservation]] = Field(default_factory=list)
```

Here's why.
At first look this looks broken -- using a mutable object as the default. But it indeed does work and it's possibly some accidental magic in Pydantic. I think it must be using deepcopy when creating objects and I recommend we don't rely on that always being true.

Reply:
thanks for pointing this out. Updated